schema/schema.json

```diff
@@ -128,6 +129,53 @@
                             }
                         }
                     },
+                    "ASCIITCPConnDetails": {
+                        "type": "object",
+                        "title": "ASCII TCP Details",
+                        "options": {
+                            "dependencies": {
+                                "connType": "ASCII TCP"
+                            },
+                            "collapsed": false
+                        },
+                        "required": [
+                            "ip",
+                            "port",
+                            "encoding",
+                            "delimiter",
+                            "keepAlive"
+
+                        ],
+                        "properties": {
+                            "ip": {
+                                "type": "string",
+                                "minLength": 3
+                            },
+                            "port": {
+                                "type": "number",
+                                "minLength": 1
+                            },
+                            "encoding": {
+                                "type": "string",
+                                "enum": [
+                                    "utf8",
+                                    "ascii",
+                                    "utf16le",
+                                    "ucs2",
+                                    "base64",
+                                    "latin1",
+                                    "binary",
+                                    "hex"
+                                ],
+                                "default": "utf8",
+                                "minLength": 1
+                            },
+                            "delimiter": {
+                                "type": "string",
+                                "minLength": 1
+                            }
```

Comment:
```suggestion
                                "minLength": 1,
                                "default": " \r\n"
                            }
```

Reply:
This is just the validation schema, I don't think defaults have any use here.